readme.md

```diff
@@ -1,4 +1,5 @@
-# Opencartdemo website login functionality Cypress automation
+# Opencartdemo website login functionality Cypress automation <!-- Title can be short and concise -->
+
```

Comment:
Title can be short and concise. You can remove the mentioned tags

Reply:
Noted and corrected.